pallets/loans/src/lib.rs

```diff
@@ -629,6 +629,31 @@ pub mod pallet {
             Ok(().into())
         }
 
+        /// Sets a new liquidation incentive percentage for `currency_id`.
+        ///
+        /// Does nothing if the provided asset is not attached to an existent incentive.
+        ///
+        /// - `currency_id`: the asset that is going to be modified.
+        #[pallet::weight(T::WeightInfo::repay_borrow_all())]
```

Comment:
Shouldn't it be `set_liquidation_incentive`? Why we are applying weight for `repay_borrow_all`?

Reply:
It is written in this PR's message:
```
Still need to perform a benchmark. The call is currently using a temporary weight from T::WeightInfo::repay_borrow_all()
```

I will now run a benchmark to create the function's weight